tests/test_unit_pto_detector_get_message_contents.py

```diff
@@ -0,0 +1,28 @@
+import os, sys
+import boto3
+import warnings
+from unittest import mock
+from unittest.mock import patch
+from moto import mock_sqs
+from pto_detector import pto_detector
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+@mock_sqs
+@patch('pto_detector.pto_detector.get_message_contents')
```

Comment:
get_message_contents is self contained. So you do not need a patch. You can simply pass it all sorts of events and make sure that it fails gracefully. If it doesn't fail gracefully, give me those events (as a test) and I can fix the method.

Reply:
@qxf2 , I have resolved your comments also, removed unittest dependency from the tests as discussed in sprint meeting. Can I share this for PR for review now?